docs/DeveloperGuide.md

```diff
@@ -1255,6 +1369,43 @@ Incorrect `remind` command to try:
 
 Expected: Error details will be shown in the status message.
 
+### Fees Calculator
+
+<div markdown="span" class="alert alert-info">:information_source: <b>Note:</b> The test cases below are designed separately and independent of each other. i.e. Test the different test cases separately.</div>
+
+1. Add the recurring lessons and makeup lessons to a student, following the test cases. <br>
+   Prerequisites: The lessons do not clash with each other and other existing lessons. If there were clashes, delete the clashing lesson(s).
```

Comment:
```suggestion
   Prerequisites: The lessons do not clash with each other and other existing lessons. If there were clashes, delete the clashing lesson(s). The student to which the lesson is to be added has no other lessons.
```
Adding this line cus ur expected result specifies that the lesson is the first lesson for the student